docs/src/main/asciidoc/logging.adoc

```diff
@@ -91,21 +91,28 @@ application POM as the following snippet illustrates.
 .Modifications to POM file to add the JSON logging extension
 [source,xml]
 ----
-<build>
   <dependencies>
     <!-- ... your other dependencies are here ... -->
     <dependency>
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-logging-json</artifactId>
     </dependency>
   </dependencies>
-</build>
 ----
 
 The presence of this extension will, by default, replace the output format configuration from the console configuration.
 This means that the format string and the color settings (if any) will be ignored.  The other console configuration items
 (including those controlling asynchronous logging and the log level) will continue to be applied.
 
+For some, it will make sense to use logging that is humanly readable (unstructured) in dev mode and JSON logging (structured) in production mode. This can be achieved using the different profiles, as shown in the following configuration. 
```

Comment:
```suggestion
For some, it will make sense to use logging that is humanly readable (unstructured) in dev mode and JSON logging (structured) in production mode. This can be achieved using different profiles, as shown in the following configuration. 
```
would be better I think.